### PR TITLE
Issue 3698

### LINKED ISSUE #3698: MIDI: Im Betrieb mit einem ESI MIDI Mate kommt es zu Notenaussetzern und Notenhängern (Local Off)
Setup: C15 -> MIDI Mate -> MIDI-Interface -> DAW -> MIDI-Interface -> MIDI Mate -> C15
OS: Mindestens seit 23-07 (bemerkt jetzt auf 23-12, nach Downgrade auf 23-07 auch dort festgestellt)
Local: Off

Es kommt vor allem dann zu Notenaussetzern wenn länger nicht gespielt (gefühlt ist es auch so, je länger nicht gespielt, desto länger dauert es bis die erste Taste wirklich klingt). Teilweise kommt es auch direkt im Anschluss an das Umstellen von Local On -> Local Off zu Notenaussetzern. Teilweise auch dann nochmal, wenn der MIDI Setup Menü-Eintrag Local verlassen wird. 

Mit der MIDI-Bridge taucht das Problem nicht auf. Insofern vielleicht nicht unbedingt ein Problem auf unserer Seite, dennoch sollten wir das näher untersuchen. 

MIDI Mate: https://www.esi-audio.de/produkte/midimateex/

### PR DESCRIPTION
**Pull request by schlenkibus**
closes #3698 by queuing active-sensing messages every 250ms

### CODE REVIEW

**Comment by schlenkibus** [comment]
@KlausStrohhaecker the setting can be found under setup -> midi settings -> assignments -> below enable high velocity

**Comment by KlausStrohhaecker** [comment]
;-)

File: projects/epc/audio-engine/src/synth/C15Synth.h [hunk 1]
```diff
@@ -89,4 +90,5 @@ class C15Synth : public Synth, public sigc::trackable
   std::mutex m_syncExternalsMutex;
   std::atomic<bool> m_quit { false };
   std::future<void> m_syncExternalsTask;
+  std::future<void> m_activeSensing;
```
Comment (hhoegelo) [inline projects/epc/audio-engine/src/synth/C15Synth.h:93]: Consider using an Expiration object that gets refreshed in queueExternalMidiOut.

File: projects/epc/audio-engine/src/synth/C15Synth.cpp [hunk 2]
```diff
@@ -191,6 +198,21 @@ void C15Synth::syncExternalsLoop()
   }
 }
 
+void C15Synth::sendActiveSensing()
```
[the remark this answers is not in the document]
Comment (hhoegelo) [reply projects/epc/audio-engine/src/synth/C15Synth.cpp:201]: daddy?

**Review by hhoegelo — APPROVED** [review]

**Pull request merged**